source/docs/hardware-reference/cancoder/index.rst

```diff
@@ -128,7 +128,7 @@ Status Light Reference
                 <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='red' offcolor='orange'></div></div></td>
                 <td>Alternate Red/Orange</td>
                 <td>Damaged Hardware.</td>
-                <td>Contact CTRE.</td>
+                <td>Use Tuner X Self Test to confirm the LEDs, then contact CTRE.</td>
```

Comment:
I would just recommend the user just check the hardware fault is tripped in self test. They might still interpret the LEDs as something incorrect.